fix/gsi/gsiparm.anl.sh

```diff
@@ -20,6 +20,7 @@ gsi_namelist="
    use_gfs_nemsio=.false.,use_gfs_ncio=.true.,reset_bad_radbc=.true.,
    netcdf_diag=${netcdf_diag},binary_diag=${binary_diag},
    l_obsprvdiag=${l_obsprvdiag},
+   lwrite_peakwt=.true.,
```

Comment:
If this is set to true by default, will it affect other applications?  If yes, we need to treat this parameter as configurable (for example, like l_obsprvdiag)

Reply:
According to the GSI v3.7 user guide, lwrite_peakwt is an option to write out the approximate pressure of the peak of the weighting function for satellite data to the radiance diagnostic files.  

In short, since the diag files are only used by enkf.x in the workflow, adding "lwrite_peakwt=.true.," in gsiparm.anl.sh should not affect other applications in the workflow.  

I have further investigated the impact with "lwrite_peakwt=.true." and with "lwrite_peakwt=.false." on the diag files generated via the GSI-observer.  Specifically, I checked the following three files from a single cycle of an RRFS-EnKF run:

- diag_conv_t_ges.2022020702.nc4
- diag_iasi_metop-b_ges.2022020702.nc4
- diag_amsua_metop-b_ges.2022020702.nc4


The results show:

- For diag_conv_t_ges.2022020702.nc4, there is no impact.  It makes sense, as lwrite_peakwt is for writing out satellite information for radiance DA in EnKF.
- For diag_iasi_metop-b_ges.2022020702.nc4, the file size increases from 4.0 MB (lwrite_peakwt=.false.) to 4.3 MB (lwrite_peakwt=.true.).  The larger file has a new variable called Press_Max_Weight_Function, which does not exist in the smaller file.  In the global attributes, iextra is changed from 0 to 1 while jextra is changed from 0 to 616.  The smaller file (lwrite_peakwt=.false.) has iextra=0 and jextra=0.  Those are all the differences.
- For diag_amsua_metop-b_ges.2022020702.nc4, the file size increases from 290 KB (lwrite_peakwt=.false.) to 300 KB (lwrite_peakwt=.true.).  Like the IASI one, using lwrite_peakwt=.true. adds variable Press_Max_Weight_Function in the diag files and changes global attributes iextra and jextra from 0 to numbers.  

Note that in conventional diag files, there are no such attributes iextra or jextra.

Reply:
Thanks, @LiaofanLin-NOAA. This is great documentation of lwrite_peakwt.